api/models/Comment.js

```diff
@@ -45,6 +57,33 @@ module.exports = bookshelf.Model.extend(Object.assign({
     return this.belongsTo(Comment).where('comments.active', true)
   },
 
+  reaction: function (userId, data) {
+    return this.userReactions(userId)
+      .then(userReactions => bookshelf.transaction(async trx => {
+        const delta = userReactions.length > 0 ? 0 : 1
+        const commentReactions = await this.get('reactions')
```

Comment:
how does .get('reactions') work when the relationship is called commentReactions above?